extensions/github1s/package.json

```diff
@@ -90,6 +91,12 @@
         "command": "github1s.open-gitpod",
         "title": "Edit files in Gitpod",
         "category": "GitHub1s"
+      },
+      {
+        "command": "github1s.open-diff-head-file",
+        "title": "Open File",
+        "category": "GitHub1s",
+        "icon": "$(go-to-file)"
```

Comment:
Do we need to register the comment here?
![image](https://user-images.githubusercontent.com/503123/112688577-51ff5c00-8e36-11eb-8bf8-8abe00a2ba5e.png)
The command will be shown on this view and it makes no action regardless it's in diff view or not.

It only works when I click this icon:
![image](https://user-images.githubusercontent.com/503123/112688693-72c7b180-8e36-11eb-855b-76a44e4fd423.png)

Reply:
Thank you so much for pointing out the problem!

we have to put the command to `contributes.commands` to indicate the `icon` and the `title`.

I found we can hide it from commandPalette by editing `contributes.menus.commandPalette` in `package.json`